src/js/utils/ui.js

```diff
@@ -21,6 +21,12 @@ const LONG_PRESS_DELAY = 500;
 let longPressTimeout;
 let lastInteractionListener;
 
+if (OS.iOS && OS.version.major < 11) {
+    const body = self.document.body;
+    // When controls are disabled iOS 10 does not dispatch media element touchstart/end events without this line
+    body.ontouchstart = body.ontouchstart || function() {};
```

Comment:
should we use `addEventLIstener` here to avoid overwriting other listeners sites might place `ontouchstart`? or does this workaround require setting the function itself

Reply:
with the `||` here we should be safe now that i think about it. anything set before this executes is preserved and anything set after will overwrite this no-op

Reply:
yep. I figured is someone else added `ontouchstart` then we don't need to.